bevrand.uitests/cypress/integration/DetailPage/details.js

```diff
@@ -0,0 +1,23 @@
+import {Then, When} from "cypress-cucumber-preprocessor/steps";
+
+When(/^I click on more details for the playlist with position (.+)$/,(number) => {
+    cy.get(`#playlistCarousel .VueCarousel-wrapper > div div:nth-child(${number})`).within(() => {
+        cy.get('#playlistName > h3')
+            .as('selectedPlayList');
+        cy.get('#linkToAllDrinks')
+            .click()
+    });
+});
+
+When(/^I click the dice icon to select the playlist$/,() => {
+    cy.get('#dicebutton').click()
```

Comment:
Before clicking, should we first assert the button is visible, enabled, and exists? Of course this counts for any click